packages/runtime/container-runtime/src/opProperties.ts

```diff
@@ -8,9 +8,12 @@ import { ISequencedDocumentMessage, ISequencedDocumentSystemMessage } from "@flu
 export const opSize = (op: ISequencedDocumentMessage): number => {
     // Some messages may already have string contents,
     // so stringifying them again will add inaccurate overhead.
-    const content = typeof op.contents === "string" ?
-        op.contents :
-        JSON.stringify(op.contents);
+    let content = "";
+    if (op.contents) {
+        content = typeof op.contents === "string" ?
+            op.contents :
+            JSON.stringify(op.contents);
```

Comment:
```suggestion
            JSON.stringify(op.contents) ?? "";
```

and this way you can drop the `let`

Reply:
Currently, if JSON.stringify returns undefined, it will still override content.